assets/js/services/metadata.js

```diff
@@ -410,6 +411,14 @@ export function deleteKeyFromObject(item) {
   return itemObj;
 }
 
+export function prepEDTFforPost(items) {
```

Comment:
No biggie, but you can set a default param for `items` (`prepEDTFforPost(items = [])`) and then don't need the ternary in the code itself.